src/vsock/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vhost-user-vsock"
+version = "0.1.0"
+authors = ["Harshavardhan Unnibhavi <harshanavkis@gmail.com>"]
+description = "This is a template for creating rust-vmm repositories."
+repository = "https://github.com/rust-vmm/crate-template"
+readme = "README.md"
+keywords = ["vhost", "vsock"]
+license = "Apache-2.0 OR BSD-3-Clause"
```

Comment:
If there's no code licensed under bsd-3-clause, how about licensing it by Apache2 only?

Reply:
The reason why we keep the dual license is so projects such QEMU can use rust-vmm crates as well. Apache 2.0 is incompatible with GPL v2. So to be able to use it, we would need to release the code under a dual license, out of which one needs to be compatible (in this case is BSD-3-Clause).